tests/unit_tests/test_lobbyconnection.py

```diff
@@ -927,6 +928,31 @@ def policy_server(event_loop):
     lobbyconnection.ladder_service.cancel_search.assert_called_with(lobbyconnection.player)
 
 
+async def test_command_matchmaker_info(lobbyconnection, ladder_service):
```

Comment:
Why do we have new test on matchmaker in python upgrade? O_o

Reply:
Hm, yea I'm not sure anymore. Maybe I just noticed it was uncovered when looking through coveralls or something.

Reply:
I think this is also a bit of a general structure refactoring now :)